scripts/launch-qemu.sh

```diff
@@ -1,5 +1,5 @@
-# SPDX-License-Identifier: MIT
 #!/bin/bash
+# SPDX-License-Identifier: MIT
```

Comment:
Why did you move this? Were you having issues with `#!/bin/bash` on the second line? If you were not facing error, let's not change this. If you were, send a separate patch changing the ordering in all four scripts. One way or the other, this should leave this commit.

Reply:
Because, that's how shebangs work. Else, this would be executed under whatever is pointed to by `/bin/sh` and not by `/bin/bash` as you would expect. How [kernel](https://github.com/torvalds/linux/blob/81ac25651a62c958bb0e074e0d4e25060ea557dd/fs/binfmt_script.c#L41) interprets it. For comparison, here is a how SPDX's are added to [kernel scripts](https://github.com/torvalds/linux/blob/master/kernel/gen_kheaders.sh#L2). [Github code search](https://github.com/search?utf8=%E2%9C%93&q=SPDX-License-Identifier+lang%3AShell+&type=code) of scripts containing SPDX.

Agree with moving it to a separate patch. will amend.

Reply:
Yep, makes sense. Looking forward to that separate patch.